frontend/pages/pings/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Stack } from '@mui/material';
+import { useTranslation } from 'next-i18next';
+import React from 'react';
+import PingCard from '~/components/Ping/PingCard';
+import genGetProps from '~/functions/genGetServerSideProps';
+import { useGetPingsQuery } from '~/generated/graphql';
+import { useSetPageName } from '~/providers/PageNameProvider';
+
+export default function Pings() {
```

Comment:
There's no obvious way to navigate to this page. I assume you get a notification upon being poked, but if you dismiss that there's no way to find this page.

Reply:
That's the idea